test/compflags/albrecht/chplenv/chplenv.chpl

```diff
@@ -12,13 +10,11 @@ writeln("CHPL_LAUNCHER=",CHPL_LAUNCHER);
 writeln("CHPL_TIMERS=",CHPL_TIMERS);
 writeln("CHPL_UNWIND=",CHPL_UNWIND);
 writeln("CHPL_MEM=",CHPL_MEM);
-writeln("CHPL_JEMALLOC=",CHPL_JEMALLOC);
```

Comment:
Removed since CHPL_JEMALLOC is now "internal" (ie not printed for --all e.g.) right?

Reply:
That is correct.